openlibrary/macros/LoanStatus.html

```diff
@@ -20,6 +20,7 @@
 
 $ user_loan = None
 $if user:
+    $ account = get_ol_account(ctx.user.key)
```

Comment:
After linking is done, there's really only one type of account -- an OL account (with an .itemname property set). The InternetArchiveAccount and OpenLibraryAccount both extend the same pre-existing Account class and provide different methods depending on whether account data should come from OL or IA (in order to auth and link accounts).

If a user is linked and logged in to OL, we can guarantee they are logged in as ctx.user.key -- i.e. /people/<username> (their OL username). This is true even if they login w/ their IA credentials, the account we fetch is still their OL `account` which has an `itemname` link set. 

From this web.ctx.key, we can fetch the user's Account by their OL username. Once we have this account, we can access their itemname. The user's `account` is necessary as the ctx.user does not have access to itemname.